app/workers/asset_manager_update_whitehall_asset_worker.rb

```diff
@@ -0,0 +1,15 @@
+class AssetManagerUpdateWhitehallAssetWorker < WorkerBase
+  sidekiq_options queue: "asset_manager_updater"
+
+  def perform(attachment_data_global_id, attributes)
+    asset_data = GlobalID::Locator.locate(attachment_data_global_id)
```

Comment:
Do we think there's risk that this may raise an exception? could an asset be attachment data be deleted during the execution of this?